src/library/scala/math/package.scala

```diff
@@ -315,13 +356,83 @@ package object math {
   /** Returns the size of an ulp of the given `Double` value.
     * @group ulp
     */
-  def ulp(x: Double): Double = java.lang.Math.ulp(x)
+  def ulp(d: Double): Double = java.lang.Math.ulp(d)
 
   /** Returns the size of an ulp of the given `Float` value.
     * @group ulp
     */
-  def ulp(x: Float): Float = java.lang.Math.ulp(x)
+  def ulp(f: Float): Float = java.lang.Math.ulp(f)
 
-  /** @group rounding */
+  /** @group exact */
   def IEEEremainder(x: Double, y: Double): Double = java.lang.Math.IEEEremainder(x, y)
+
+  // -----------------------------------------------------------------------
+  // exact functions
+  // -----------------------------------------------------------------------
+
+  /**
+    * @group exact
+    */
+  def addExact(x: Int, y: Int): Int = java.lang.Math.addExact(x, y)
+
+  /**
+    * @group exact
+    */
+  def addExact(x: Long, y: Long): Long = java.lang.Math.addExact(x, y)
+
+  /**
+    * @group exact
+    */
+  def subtractExact(x: Int, y: Int): Int = java.lang.Math.subtractExact(x, y)
+
+  /**
+    * @group exact
+    */
+  def subtractExact(x: Long, y: Long): Long = java.lang.Math.subtractExact(x, y)
+
+  /**
+    * @group exact
+    */
+  def multiplyExact(x: Int, y: Int): Int = java.lang.Math.multiplyExact(x, y)
+
+  /**
+    * @group exact
+    */
+  def multiplyExact(x: Long, y: Long): Long = java.lang.Math.multiplyExact(x, y)
+
+  /**
+    * @group exact
+    */
+  def incrementExact(a: Int): Int = java.lang.Math.incrementExact(a)
```

Comment:
`a` is carried over from the Java 8 names. Maybe the Java engineers are working to some hidden conventation but I can't figure it out. For example in Java

    log(double a)
    log10(double a)
    log1p(double x)
    toIntExact(long value)

Possible there are some conventions in the world of floating point arithmetic at play here.

Overall I would just stick with `x` unless there is reason not too.